exporters/metrics/src/main/java/com.google.cloud.opentelemetry.metric/MetricExporter.java

```diff
@@ -131,18 +129,16 @@ public CompletableResultCode export(Collection<MetricData> metrics) {
         continue;
       }
 
-      Metric metric = mapMetric(metricPoint, descriptor.getType());
       Point point = mapPoint(metricData, metricPoint, updateKey, lastUpdatedTime);
       if (point == null) {
         continue;
       }
 
       allTimesSeries.add(
           TimeSeries.newBuilder()
-              .setMetric(metric)
-              .addPoints(point)
-              .setResource(MonitoredResource.newBuilder().build())
```

Comment:
The lack of the resource here makes the mock server crash, so either the mock server needs to be modified, or the resource mapping needs to be done properly, before the end to end tests can pass.